recipe/meta.yaml

```diff
@@ -1,61 +1,279 @@
-{% set version = "3.6.0rc1" %}
-{% set sha256 = "e80bc33d09f1f3c06ed760c921e7e279479325b7db372afe95fa05deabad4db9" %}
+{% set version = "3.9.0" %}
+{% set dev = "rc1" %}
+{% set linkage_nature = os.environ.get('PY_INTERP_LINKAGE_NATURE', '') %}
+{% set debug = os.environ.get('PY_INTERP_DEBUG', '') %}
+{% if linkage_nature != '' %}
+  {% set linkage_nature = "_" ~ linkage_nature %}
+{% endif %}
+{% if debug != '' %}
+  {% set debug = "_d" ~ debug %}
+{% endif %}
 
 package:
-  name: python
+  name: python-split
   version: {{ version }}
 
 source:
-  fn: Python-{{ version }}.tgz
-  url: https://www.python.org/ftp/python/3.6.0/Python-{{ version }}.tgz
-  sha256: {{ sha256 }}
+  - url: https://www.python.org/ftp/python/{{ version }}/Python-{{ version }}{{ dev }}.tar.xz
+    # md5 from: https://www.python.org/downloads/release/python-385/
+    md5: 835f3cb29e4065b9a2dea1c7b9bfc37f
+    patches:
+{% if 'conda-forge' not in channel_targets %}
+      - patches/0001-Add-Anaconda-Distribution-version-logic.patch
+{% endif %}
+      - patches/0002-Darwin-Add-arch-x86_64-to-UnicCCompiler-ld_args.patch
+      - patches/0003-Win32-Change-FD_SETSIZE-from-512-to-2048.patch
+      - patches/0004-Win32-distutils-Add-support-to-cygwinccompiler-for-V.patch
+      - patches/0005-Do-not-pass-g-to-GCC-when-not-Py_DEBUG.patch
+      - patches/0006-Support-cross-compiling-byte-code.patch
+      - patches/0007-Win32-Fixes-for-Windows-GCC-interop-needed-by-RPy2-a.patch
+      - patches/0008-Darwin-Look-in-sysroot-usr-lib-include-if-sysroot-is.patch
+      - patches/0009-runtime_library_dir_option-Use-1st-word-of-CC-as-com.patch
+      - patches/0010-Win32-Do-not-download-externals.patch
+      - patches/0011-Add-support-for-_CONDA_PYTHON_SYSCONFIGDATA_NAME-if-.patch
+      - patches/0012-Fix-find_library-so-that-it-looks-in-sys.prefix-lib-.patch
+      - patches/0013-Disable-new-dtags-in-unixccompiler.py.patch
+      - patches/0014-Fix-cross-compilation-on-Debian-based-distros.patch
+      # - patches/0015-Disable-registry-lookup-unless-CONDA_PY_ALLOW_REG_PA.patch
```

Comment:
@mingwandroid this is inside a `if (!Py_IgnoreEnvironmentFlag)` now. Does that solve our problem or do we still need to refresh this patch.

Reply:
I highly recommend keeping this. Without it, installers will fail, for example. We don't set `Py_IgnoreEnvironmentFlag` and until we do (if we want to) and get this enforced in all the places we need to.

Reply:
Actually I am not sure. I mean, this is our own var that some may have used.. if no one is using it then yeah, let's drop it.